_data/chains/eip155-88808.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Oragon Network Testnet",
+  "chain": "Oragon Testnet",
+  "icon": "oragonnetwork",
+  "rpc": ["https://rpc.oragon.network/index.php/node/ethereum_api"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Oragon",
+    "symbol": "ORGN",
+    "decimals": 18
+  },
+  "infoURL": "https://oragon.network",
+  "shortName": "oragonnetwork",
+  "chainId": 88808,
+  "networkId": 88808,
+  "explorers": [
+    {
+      "name": "Oragon Network Explorer",
```

Comment:
If this will be use once you have mainnet launch, prefer using ORGN Explorer Testnet as shown on url below